### ISSUE TITLE
Ctrl+Z (Undo) stops working

### ISSUE THREAD

**Issue by Zrit234** [opentoonz/opentoonz#4905]
### Description

After commands CTRL+C and CTRL+V Ctrl+Z stop working correctly. If I use it on Level strip, it works correctly, but when I combine Ctrl+C and Ctrl+V and then try to use Ctrl+Z on Xsheet, it doesn't work(

### Steps to Reproduce

1. copy and paste any part of drawing 
2. Try to use Ctrl+Z
3. It doesn't work correctly, it deletes levels and other things

### Expected Behavior

I draw the line, and I want to cancel it to redraw, but I can't

### OpenToonz Version

1.7

### Operating System

Windows

### GPU

NVIDIA

### Graphics Tablet

None

**Comment by RodneyBaker** [comment]
Download the latest nightly release as that has the fix from 11 May that should resolve that.
That specifica fix is the third nightly down on the list but you might as well get the other two fixes since then as well.

Link to nightlies:  https://github.com/opentoonz/opentoonz_nightlies/releases

**Comment by heyBrettJosh** [comment]
I have the same issue with the control Z. 
Which file should I download to fix it?

**Comment by RodneyBaker** [comment]
I had a similar issue with Undo (prior to the fix that is now in the nightly release).

What I did to resolve the problem was temporarily reassign the shortcut key from Control + Z to just Z.
"Set Key" (Set Keyframe) is assigned the shortcut key Z by default but I don't use that nearly as often as I do Undo so that worked well for me.  I did have to remember to switch the shortcuts back later though as the use of Control Z no longer worked for anything!  ;)

@heyBrettJosh If you are on Windows you'll want the .exe file.
Direct link:  https://github.com/opentoonz/opentoonz_nightlies/releases/download/nightly_0c59f70c2d5b05e7f8b668ea7963dc49764ae274/OpenToonzSetup.exe
